### PR TITLE
Add MessageFeedServlet.java

### PR DESCRIPTION
**Pull request by minhphu20**
Add a new servlet called MessageFeedServlet.java

### CODE REVIEW

File: src/main/java/com/google/codeu/servlets/MessageFeedServlet.java [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    public void init() {
+        datastore = new Datastore();
+    }
+ 
+    /**
+     * Responds with a JSON representation of Message data for all users.
+     */
+    @Override
+    public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+
```
Comment (emto) [inline src/main/java/com/google/codeu/servlets/MessageFeedServlet.java]: unnecessary newline

File: src/main/java/com/google/codeu/data/Datastore.java [hunk 2]
```diff
@@ -53,18 +53,42 @@ public void storeMessage(Message message) {
...
             .setFilter(new Query.FilterPredicate("user", FilterOperator.EQUAL, user))
             .addSort("timestamp", SortDirection.DESCENDING);
     PreparedQuery results = datastore.prepare(query);
 
+    return getMessages(results);
+  }
+
+  /**
+   * Fetches the messages of all users.
+   *
```
Comment (emto) [inline src/main/java/com/google/codeu/data/Datastore.java:66]: or empty list if no one

File: src/main/java/com/google/codeu/data/Datastore.java [hunk 3]
```diff
@@ -76,7 +100,7 @@ public void storeMessage(Message message) {
         e.printStackTrace();
       }
     }
-
     return messages;
   }
+
```
Comment (emto) [inline src/main/java/com/google/codeu/data/Datastore.java]: remove line

Comment (minhphu20) [reply src/main/java/com/google/codeu/data/Datastore.java]: Removed!

**Review by emto — APPROVED** [review]

**Pull request merged**